tests/test_hooks/test_asg_scheduled_actions.py

```diff
@@ -96,12 +100,23 @@ def test_run_with_resume_argument(self, mock_find_autoscaling_groups):
                     ]
                 }
             )
+        mock_warn.assert_called_once_with(
+            "{0}The asg_scheduled_actions hook has been deprecated, and will "
+            "be removed in a later version of Sceptre. Use the "
+            "asg_scaling_processes hook instead. Example: "
+            "!asg_scaling_processes <suspend|resume>::ScheduledActions{1}"
+            .format(Fore.YELLOW, Style.RESET_ALL),
```

Comment:
I think it would be convenient (and would fix the comma splices) if you made this a variable and used them throughout the tests.